include/seqan3/alignment/pairwise/all.hpp

```diff
@@ -272,3 +262,29 @@
  *  - [lecture script - pairwise alignment](https://www.mi.fu-berlin.de/en/inf/groups/abi/teaching/lectures/lectures_past/WS0910/V___Algorithmen_und_Datenstrukturen/scripts/alignment.pdf)\n
  *  - alignment
  */
+
+#pragma once
+
+#include <seqan3/alignment/pairwise/align_pairwise.hpp>
+#include <seqan3/alignment/pairwise/align_result_selector.hpp>
+#include <seqan3/alignment/pairwise/alignment_algorithm.hpp>
+#include <seqan3/alignment/pairwise/alignment_configurator.hpp>
+#include <seqan3/alignment/pairwise/alignment_result.hpp>
+#include <seqan3/alignment/pairwise/detail/alignment_algorithm_state.hpp>
+#include <seqan3/alignment/pairwise/detail/concept.hpp>
+#include <seqan3/alignment/pairwise/detail/pairwise_alignment_algorithm.hpp>
+#include <seqan3/alignment/pairwise/detail/pairwise_alignment_algorithm_banded.hpp>
+#include <seqan3/alignment/pairwise/detail/policy_affine_gap_recursion.hpp>
+#include <seqan3/alignment/pairwise/detail/policy_affine_gap_recursion_banded.hpp>
+#include <seqan3/alignment/pairwise/detail/policy_affine_gap_with_trace_recursion.hpp>
+#include <seqan3/alignment/pairwise/detail/policy_affine_gap_with_trace_recursion_banded.hpp>
+#include <seqan3/alignment/pairwise/detail/policy_alignment_matrix.hpp>
+#include <seqan3/alignment/pairwise/detail/policy_alignment_result_builder.hpp>
+#include <seqan3/alignment/pairwise/detail/policy_optimum_tracker.hpp>
+#include <seqan3/alignment/pairwise/detail/policy_optimum_tracker_simd.hpp>
+#include <seqan3/alignment/pairwise/detail/policy_scoring_scheme.hpp>
```

Comment:
As far as I am aware, the `all.hpp` should not include `detail` files.

Reply:
yes. I hope that's not too hard to avoid?